rotkehlchen/tests/unit/test_convex.py

```diff
@@ -0,0 +1,25 @@
+from rotkehlchen.chain.ethereum.modules import Convex
+from rotkehlchen.chain.ethereum.types import string_to_ethereum_address
+from rotkehlchen.user_messages import MessagesAggregator
+
+
+def test_convex_balances(database, ethereum_manager, inquirer):
+    msg_aggregator = MessagesAggregator()
+    inquirer.inject_ethereum(ethereum_manager)
+    convex = Convex(
+        database=database,
+        ethereum_manager=ethereum_manager,
+        premium=None,
+        msg_aggregator=msg_aggregator,
+    )
+    # addr1 and addr2 test staking of 2 addresses in the same pool (cvxCRV/CRV in this case)
+    addr1 = string_to_ethereum_address('0x0633F1692Dab82F6F253eF571C769861CF90ecb6')
+    addr2 = string_to_ethereum_address('0xC49Eb99c132795F74b3d6f71B2374dC35015d473')
+    # addr3 tests staking in multiple pools
+    addr3 = string_to_ethereum_address('0xd6ffc26607A60A0c5383533fd53F7C47c26D62D4')
+    user_addresses = [addr1, addr2, addr3]
+    found_balances = convex.get_balances(addresses=user_addresses)
+
+    assert len(found_balances[addr1]) == 2
+    assert len(found_balances[addr2]) == 2
```

Comment:
What is this testing exactly? Is that the address has two rewards? Also will this test break if the addresses withdraw? If that is the case you should change the approach here

Reply:
Improved the test